testsuite/objects/__init__.py

```diff
@@ -86,15 +93,21 @@ class Value(ABCValue):
     value: JSONValues
 
 
+@dataclass
+class _AuthJson:
+    authJSON: str  # pylint: disable=invalid-name
+
+
 @dataclass
 class ValueFrom(ABCValue):
     """Dataclass for dynamic Value. It contains reference path to existing value in AuthJson."""
 
-    authJSON: str  # pylint: disable=invalid-name
+    valueFrom: _AuthJson | str  # pylint: disable=invalid-name
```

Comment:
And if so, why keeping 'str' and '__post_init__'? (I see why, however the tradeoff to remove it doesn't seem to be that big)